pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -246,7 +281,21 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 	}
 
 	// The suite already exists, should we update?
-	if suiteNeedsUpdate(&suite, &found) {
+	if r.suiteNeedsUpdate(&suite, &found, instance) {
+		// If the underlying ScanSetting used in the ScanSettingBinding
+		// has changed, then we need to make sure we copy those new
+		// settings to the ComplianceSuite.
+		if instance.SettingsRef != nil {
+			scanSettings := &compliancev1alpha1.ScanSetting{}
+			err := r.Client.Get(context.TODO(), types.NamespacedName{Name: instance.SettingsRef.Name, Namespace: instance.Namespace}, scanSettings)
+			if err != nil {
+				// What should happen here?
+			}
+			suite.Spec.ComplianceSuiteSettings.Suspend = scanSettings.ComplianceSuiteSettings.Suspend
+			suite.Spec.ComplianceSuiteSettings.Schedule = scanSettings.ComplianceSuiteSettings.Schedule
+		}
+
```

Comment:
same as below, I think we should already have the latest scansetting in the suite because https://github.com/ComplianceAsCode/compliance-operator/pull/396/files#diff-c647a52d3b0966655cdd2384a18bfdaa5e8d694df700395aabba380140b89ef2R211

Reply:
Good catch - fixed.